science/HDF5-External-Filter-Plugins/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           muniversal 1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.0
+
+github.setup        nexusformat HDF5-External-Filter-Plugins 0.1.0 v
+categories          science
+platforms           darwin
+license             BSD MIT
+maintainers         {@eborisch eborisch} openmaintainer
+description         Provides dynamically loadable (compression) filters for HDF5
+long_description    ${description}. Filters for bitshuffle, bzip2, and lz4 are \
+                    included. See also \
+                    http://www.hdfgroup.org/services/filters.html for \
+                    more information.
+
+checksums \
+    rmd160  6724d20763725fe59d04329d5069e2943eec99cb \
+    sha256  3da58e714b94d2e1bffbf52a6b64952733874ef383c23a1ad08cd01cc69f28f9 \
+    size    943586
+
+variant optimized {
+    # Bitshuffle code has sse and AVX implementations
+    configure.cxxflags-append   -march=native
+}
+
+depends_lib         port:hdf5 \
```

Comment:
When using portgroups, it best practice to use `depends_lib-append` to not overwrite portgroup dependencies.

Reply:
Makes no difference here, but, as you say, it is best practice. Will do.